src/bin/deleteSpecificVersion.ts

```diff
@@ -0,0 +1,55 @@
+import { SupabaseClient } from '@supabase/supabase-js';
```

Comment:
@riderx I abstracted this into a separate file, but i think it would be better to move this to something like /src/api or /src/service instead. 

It seems a bit more abstraction would be useful, as several commands can use the same api service.

Reply:
Moved this into /src/api namespace.